content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Mengoperasikan klaster etcd untuk Kubernetes
+content_type: task
+---
+
+<!-- overview -->
+
+{{< glossary_definition term_id="etcd" length="all" prepend="etcd adalah ">}}
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Prerequisites
+
+* Jalankan etcd sebagai klaster dari anggota ganjil.
```

Comment:
can be better rephrased for better readability:
```suggestion
* Jalankan etcd sebagai klaster dimana anggotanya berjumlah ganjil.
```

Reply:
Okay, I think `Jalankan etcd sebagai klaster dimana anggotanya berjumlah ganjil.` has better readability.